packages/expo-secure-store/src/SecureStore.ts

```diff
@@ -69,10 +72,13 @@ export type SecureStoreOptions = {
    * accessing data stored in SecureStore.
    * - Android: Equivalent to [`setUserAuthenticationRequired(true)`](https://developer.android.com/reference/android/security/keystore/KeyGenParameterSpec.Builder#setUserAuthenticationRequired(boolean))
    *   (requires API 23).
-   * - iOS: Equivalent to [`kSecAccessControlBiometryCurrentSet`](https://developer.apple.com/documentation/security/secaccesscontrolcreateflags/ksecaccesscontrolbiometrycurrentset/).
+   * - iOS: Equivalent to [`biometryCurrentSet`](https://developer.apple.com/documentation/security/secaccesscontrolcreateflags/2937192-biometrycurrentset).
    * Complete functionality is unlocked only with a freshly generated key - this would not work in tandem with the `keychainService`
    * value used for the others non-authenticated operations.
    *
+   * This option works slightly differently across platforms: On iOS, the user is prompted to authenticate only when reading or updating an existing value (not when creating a new one).
+   * On Android, user authentication is required for all operations.
+   *
```

Comment:
```suggestion
   * This option works slightly differently across platforms: On Android, user authentication is required for all operations. 
   * On iOS, the user is prompted to authenticate only when reading or updating an existing value (not when creating a new one).
   *
```

Let's use Platform order when referencing [multiple platforms as per our writing style guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#referencing-android-ios-and-web).